src/altinn-app-frontend/src/features/expressions/validation.ts

```diff
@@ -62,17 +62,17 @@ function validateFunctionArg(
   const expectedType = argTypeAt(func, idx);
   const actualType = actual[idx];
   if (expectedType === undefined) {
-    addError(ctx, [...path, `[${idx}]`], ValidationErrorMessage.ArgUnexpected);
+    addError(ctx, [...path, `[${idx + 1}]`], ValidationErrorMessage.ArgUnexpected);
   } else {
     const targetType = ExprTypes[expectedType];
 
     if (actualType === undefined) {
       if (targetType.nullable) {
         return;
       }
-      addError(ctx, [...path, `[${idx}]`], ValidationErrorMessage.ArgWrongType, expectedType, 'null');
+      addError(ctx, [...path, `[${idx + 1}]`], ValidationErrorMessage.ArgWrongType, expectedType, 'null');
     } else if (!targetType.accepts.includes(actualType)) {
-      addError(ctx, [...path, `[${idx}]`], ValidationErrorMessage.ArgWrongType, expectedType, 'null');
+      addError(ctx, [...path, `[${idx + 1}]`], ValidationErrorMessage.ArgWrongType, expectedType, 'null');
```

Comment:
The first value is always the function name, so this validation code misplaced the error message one value early. Fixing that at the same time.